src/config_tools/config_tools.h

```diff
@@ -106,28 +106,43 @@ void ValidateDtypeConfig(TritonJson::Value &io_object, const std::string &name,
 /**
  * @brief Auto-fills `config_io`'s dimensions field with value `model_io_shape`.
  * `config` must be a top-level TritonJson object containing `config_io`
+ *
+ * If `add_batch_dim` is set to true, the model_io_shape will be prepended with -1
+ * to extend it with the batch dimension.
```

Comment:
There's a reference to `add_batch_dim`, but there's no such argument. Did you mean `batched_model`?

Reply:
fixed